release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1372,12 +1372,28 @@ In {product-title} {product-version}, {olmv1-first} changes its default upgrade
 For more information, see xref:../release_notes/ocp-4-16-release-notes.adoc#ocp-4-16-olmv1-legacy-olm-upgrade-edges_release-notes[Support for {olmv0} upgrade edges in {olmv1} (Technology Preview)].
 
 [discrete]
-[id="ocp-4-16-builder-sa"]
-=== The builder service account is no longer created if the Build cluster capability is disabled
+[id="ocp-4-16-dal12-region_{context}"]
+=== Dal12 region added
 
-With this release, if you disable the `Build` cluster capability, the `builder` service account and its corresponding secrets are no longer created.
+With this release, the `dal12` region has been added to the Power VS Installer.
 
-For more information, see xref:../installing/cluster-capabilities.adoc#build-config-capability_cluster-capabilities[Build capability].
+[discrete]
+[id="ocp-4-16-power-vs-regions_{context}"]
+=== Regions added to power virtual server
+
+This release introduces the ability to deploy to new Power virtual server (VS) regions `osa21`, `syd04`, `lon06`, and `sao01`.
+
+[discrete]
+[id="ocp-4-16-power-vs-capi_{context}"]
+=== Power virtual server cluster API updated to 0.8.0
+
+With this release, the PowerVS CAPI has been updated to version 0.8.0.
+
+[discrete]
+[id="ocp-4-16-debug-guid_{context}"]
+=== Additional debugging statements for ServiceInstanceNameToGUID
+
+With this release, additional debugging statements have been added to the `ServiceInstanceNameToGUID` function.
```

Comment:
I would use "were added". The tense advise in IBM SG is to keep it as simple as possible https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense